pkg/converter/policy_converter_test.go

```diff
@@ -204,11 +207,13 @@ var _ = Describe("PolicyConverter", func() {
 			Expect(len(pol.(api.Policy).Spec.EgressRules)).To(Equal(0))
 		})
 
-		var policyType api.PolicyType = "ingress"
-		// Check that Types field exists and has only 'ingress'
-		It("should return calico policy with ingress type", func() {
-			Expect(len(pol.(api.Policy).Spec.Types)).To(Equal(1))
-			Expect(pol.(api.Policy).Spec.Types[0]).To(Equal(policyType))
+		var policyTypeIngress api.PolicyType = "ingress"
+		var policyTypeEgress api.PolicyType = "egress"
+		// Check that Types field exists and has both 'ingress' and 'egress'
+		It("should return calico policy with ingress and egress type", func() {
```

Comment:
Same here - this shouldn't change. Might be a bug in our conversion code.

Reply:
Looks like k8s.NetworkPolicyToPolicy() is doing the right thing, but someone after this is taking an empty Types []string and changing it to ingress+egress.

Reply:
Think we tracked this down to the UTs not sending in realistic data.

@ozdanborne @bcreane could you guys coordinate to make sure this case is tested in an FV so that we know it's doing the right thing end-to-end?

Reply:
For these tests, let's just hard-code the data to say "Ingress"